_pages/fsa/dynamicforms/core/basic-form-validations.md

```diff
@@ -0,0 +1,285 @@
+---
+title: Basic Form Validations
+---
+
+## Overview
+Dynamic Forms library provides some basic validator functions by default. This page will explain how to use, override or add validations in Dynamic Forms library for end customers. Following things should be covered:
```

Comment:
"This **page** will explain" => "This **section** will explain", maybe? Since these may not be separate pages.

Reply:
It is page.